chatkit-core/src/main/kotlin/com/pusher/chatkit/users/api/UserSubscriptionDispatcher.kt

```diff
@@ -48,4 +53,24 @@ internal class UserSubscriptionDispatcher(
                 dispatcher(RoomDeleted(event.roomId))
         }
     }
+
+    override fun onSubscribe() {
+        // Not yet implemented
+    }
+
+    override fun onOpen(headers: Headers) {
```

Comment:
likewise: https://github.com/pusher/chatkit-android/pull/202/files#r394384654

Reply:
Probably not needed, I'll drop them.